### PR TITLE
#158487956 Integrate TravisCI

### PR DESCRIPTION
**Pull request by Stylll**
#### What does this PR do?
Integrates TravisCI and adds readme badge.

#### Description of Task to be completed?
- Create travis config file to integrate app with TravisCI
- Add build badge to readme file

#### How should this be manually tested?
- Check for the .travis.yml file in the root of the folder.
- Check the readme file to view the TravisCI badge.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#158487956

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

File: test/article/model.js [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
+//                     done();
+//                 });
+//             });
+//         });
+
+//         afterEach(function(done) {
+//             done();
+//         });
+//     });
+// });
```
Comment (VictoriaOffoma) [inline test/article/model.js]: Please add newline

File: config/config.js [hunk 2]
```diff
@@ -4,4 +4,4 @@ var _ = require('underscore');
 
 module.exports = _.extend(
     require(__dirname + '/../config/env/all.js'),
-    require(__dirname + '/../config/env/' + process.env.NODE_ENV + '.json') || {});
\ No newline at end of file
+    require(__dirname + '/../config/env/' + process.env.NODE_ENV + '.js') || {});
```
Comment (VictoriaOffoma) [inline config/config.js]: Please add a newline.

File: test/article/model.js (hunk 1 above)
Comment (Stylll) [reply test/article/model.js]: Newline added

File: config/config.js (hunk 2 above)
Comment (Stylll) [reply config/config.js]: Newline added.

File: config/config.js [hunk 3]
```diff
@@ -4,4 +4,5 @@ var _ = require('underscore');
 
 module.exports = _.extend(
     require(__dirname + '/../config/env/all.js'),
-    require(__dirname + '/../config/env/' + process.env.NODE_ENV + '.js') || {});
\ No newline at end of file
+    require(__dirname + '/../config/env/' + process.env.NODE_ENV + '.js') || {});
+
```
Comment (VictoriaOffoma) [inline config/config.js]: hey, one escaped you.

Comment (Stylll) [reply config/config.js]: Line 8 is empty. It can serve as a new line right ?. Is there something I'm missing ?.

**Pull request merged**